tests/test_core.py

```diff
@@ -552,20 +553,20 @@ def test_absolute_filename():
     assert frame_info.filename == full_filename
 
 
-def test_executing_style_defs():
+@pytest.mark.parametrize("expected",
```

Comment:
parametrize seems like the wrong approach here. This isn't a family of related tests, it's a bunch of assertions about a single string.